modules/Bio/EnsEMBL/Production/Pipeline/FASTA/Base.pm

```diff
@@ -33,10 +33,25 @@ sub fasta_path {
 sub old_path {
   my ($self, $species, $directory) = @_;
   my $base = $self->param('ftp_dir');
+  my $eg = $self->param('eg');
```

Comment:
Introducing this parameter seems unnecessary; we're trying to erode the EG brand, after all. Why not just test if `$self->division` is 'vertebrates', and use that to determine the url path?

Reply:
I was using the `eg` flag because it's used consistently in this pipeline but you are right, not using it will be one less line to update when we get rid of it in the future.